install/kubernetes/helm/istio/charts/mixer/templates/istio-mixer.yaml

```diff
@@ -767,3 +802,55 @@ spec:
     - tcpbytesent.metric
     - tcpbytereceived.metric
 ---
+apiVersion: "config.istio.io/v1alpha2"
+kind: kubernetesenv
+metadata:
+  name: handler
+  namespace: {{$istio_namespace}}
+spec:
+  # when running from mixer root, use the following config after adding a
```

Comment:
this comment probably isn't useful for people installing via helm.

Reply:
D